h5io/_h5io.py

```diff
@@ -66,17 +66,26 @@ def write_hdf5(fname, data, overwrite=False, compression=4,
     title : str
         The top-level directory name to use. Typically it is useful to make
         this your package name, e.g. ``'mnepython'``.
+    update_title : bool
+        If True, override value for title (if title exists). If False and title
+        exists and error will be raised.
     """
     h5py = _check_h5py()
-    if op.isfile(fname) and not overwrite:
-        raise IOError('file "%s" exists, use overwrite=True to overwrite'
-                      % fname)
+    mode = 'a'
+    if op.isfile(fname) and overwrite:
+        mode = 'w'
     if not isinstance(title, string_types):
         raise ValueError('title must be a string')
     comp_kw = dict()
     if compression > 0:
         comp_kw = dict(compression='gzip', compression_opts=compression)
-    with h5py.File(fname, mode='w') as fid:
+    with h5py.File(fname, mode=mode) as fid:
+        if title in fid:
+            if not update_title:
+                raise IOError('title %s already exists, use update_title=True'
+                              'to override')
+            else:
```

Comment:
this doesn't actually need to be in an `else` block since the one above raises an error